src/DigitalSubscriberAppBanner.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from "react";
+import { DigitalSubscriberAppBanner } from "./DigitalSubscriberAppBanner";
+import { StorybookWrapper } from './utils/StorybookWrapper';
+
+export default {
+  component: DigitalSubscriberAppBanner,
+  title: "Components/DigitalSubscriberAppBanner",
+};
+
+export const defaultStory = (): ReactElement => {
+  return (
+    <StorybookWrapper>
+        <DigitalSubscriberAppBanner
+            onButtonClick={() => {
```

Comment:
I'd consider logging the button ID here as well, but not urgent and no harm here. We can add it later if we want to do that.

Reply:
Yes, good idea - I've implemented now.